### PR TITLE
plasma-infra(new-hope): Migrate to Storybook 7

### PR DESCRIPTION
**Pull request by Yakutoc**
## Release Notes

Переход на `Storybook@latest(7.5.3)`:

### What/why Changed

[From version 6.5.x to 7.0.0](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#from-version-65x-to-700)

- [x] rewrite `.stories` to CSF 3 (Более подробное описание - #833)
- [x] [rewrite `main.ts`](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#esm-format-in-mainjs)
- [x] [rewrite `preview.ts` ](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#default-export-in-previewjs)

<details>
  <summary>:baby_chick: Download canary assets:</summary>
  <blockquote>
    <details>
      <summary>borderRadius</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_default_react-native--canary.845.6859005179.ts'>borderRadius_default_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_b2c_react-native--canary.845.6859005179.ts'>borderRadius_plasma_b2c_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_web_react-native--canary.845.6859005179.ts'>borderRadius_plasma_web_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberHealth_react-native--canary.845.6859005179.ts'>borderRadius_sberHealth_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_business_react-native--canary.845.6859005179.ts'>borderRadius_sbermarket_business_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_metro_react-native--canary.845.6859005179.ts'>borderRadius_sbermarket_metro_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_react-native--canary.845.6859005179.ts'>borderRadius_sbermarket_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_selgros_react-native--canary.845.6859005179.ts'>borderRadius_sbermarket_selgros_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_wlbusiness_react-native--canary.845.6859005179.ts'>borderRadius_sbermarket_wlbusiness_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberonline_react-native--canary.845.6859005179.ts'>borderRadius_sberonline_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberprime_react-native--canary.845.6859005179.ts'>borderRadius_sberprime_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_stylesSalute_react-native--canary.845.6859005179.ts'>borderRadius_stylesSalute_react-native--canary.845.6859005179.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>color</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_ios-swift--canary.845.6859005179.swift'>color_default_ios-swift--canary.845.6859005179.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_kotlin--canary.845.6859005179.kt'>color_default_kotlin--canary.845.6859005179.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_react-native--canary.845.6859005179.ts'>color_default_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_xml--canary.845.6859005179.xml'>color_default_xml--canary.845.6859005179.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_ios-swift--canary.845.6859005179.swift'>color_plasma_b2c_ios-swift--canary.845.6859005179.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_kotlin--canary.845.6859005179.kt'>color_plasma_b2c_kotlin--canary.845.6859005179.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_react-native--canary.845.6859005179.ts'>color_plasma_b2c_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_xml--canary.845.6859005179.xml'>color_plasma_b2c_xml--canary.845.6859005179.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_ios-swift--canary.845.6859005179.swift'>color_plasma_web_ios-swift--canary.845.6859005179.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_kotlin--canary.845.6859005179.kt'>color_plasma_web_kotlin--canary.845.6859005179.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_react-native--canary.845.6859005179.ts'>color_plasma_web_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_xml--canary.845.6859005179.xml'>color_plasma_web_xml--canary.845.6859005179.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.845.6859005179.swift'>color_sberHealth_ios-swift--canary.845.6859005179.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.845.6859005179.kt'>color_sberHealth_kotlin--canary.845.6859005179.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.845.6859005179.ts'>color_sberHealth_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_xml--canary.845.6859005179.xml'>color_sberHealth_xml--canary.845.6859005179.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.845.6859005179.swift'>color_sbermarket_business_ios-swift--canary.845.6859005179.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.845.6859005179.kt'>color_sbermarket_business_kotlin--canary.845.6859005179.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.845.6859005179.ts'>color_sbermarket_business_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_xml--canary.845.6859005179.xml'>color_sbermarket_business_xml--canary.845.6859005179.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.845.6859005179.swift'>color_sbermarket_ios-swift--canary.845.6859005179.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.845.6859005179.kt'>color_sbermarket_kotlin--canary.845.6859005179.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.845.6859005179.swift'>color_sbermarket_metro_ios-swift--canary.845.6859005179.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.845.6859005179.kt'>color_sbermarket_metro_kotlin--canary.845.6859005179.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.845.6859005179.ts'>color_sbermarket_metro_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_xml--canary.845.6859005179.xml'>color_sbermarket_metro_xml--canary.845.6859005179.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.845.6859005179.ts'>color_sbermarket_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.845.6859005179.swift'>color_sbermarket_selgros_ios-swift--canary.845.6859005179.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.845.6859005179.kt'>color_sbermarket_selgros_kotlin--canary.845.6859005179.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.845.6859005179.ts'>color_sbermarket_selgros_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_xml--canary.845.6859005179.xml'>color_sbermarket_selgros_xml--canary.845.6859005179.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.845.6859005179.swift'>color_sbermarket_wlbusiness_ios-swift--canary.845.6859005179.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.845.6859005179.kt'>color_sbermarket_wlbusiness_kotlin--canary.845.6859005179.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.845.6859005179.ts'>color_sbermarket_wlbusiness_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_xml--canary.845.6859005179.xml'>color_sbermarket_wlbusiness_xml--canary.845.6859005179.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_xml--canary.845.6859005179.xml'>color_sbermarket_xml--canary.845.6859005179.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.845.6859005179.swift'>color_sberonline_ios-swift--canary.845.6859005179.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.845.6859005179.kt'>color_sberonline_kotlin--canary.845.6859005179.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.845.6859005179.ts'>color_sberonline_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_xml--canary.845.6859005179.xml'>color_sberonline_xml--canary.845.6859005179.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.845.6859005179.swift'>color_sberprime_ios-swift--canary.845.6859005179.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.845.6859005179.kt'>color_sberprime_kotlin--canary.845.6859005179.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.845.6859005179.ts'>color_sberprime_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_xml--canary.845.6859005179.xml'>color_sberprime_xml--canary.845.6859005179.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_ios-swift--canary.845.6859005179.swift'>color_stylesSalute_ios-swift--canary.845.6859005179.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_kotlin--canary.845.6859005179.kt'>color_stylesSalute_kotlin--canary.845.6859005179.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_react-native--canary.845.6859005179.ts'>color_stylesSalute_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_xml--canary.845.6859005179.xml'>color_stylesSalute_xml--canary.845.6859005179.xml</a><br>
      </blockquote>
    </details>
    <details>
      <summary>shadow</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_default_react-native--canary.845.6859005179.ts'>shadow_default_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_b2c_react-native--canary.845.6859005179.ts'>shadow_plasma_b2c_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_web_react-native--canary.845.6859005179.ts'>shadow_plasma_web_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberHealth_react-native--canary.845.6859005179.ts'>shadow_sberHealth_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_business_react-native--canary.845.6859005179.ts'>shadow_sbermarket_business_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_metro_react-native--canary.845.6859005179.ts'>shadow_sbermarket_metro_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.845.6859005179.ts'>shadow_sbermarket_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_selgros_react-native--canary.845.6859005179.ts'>shadow_sbermarket_selgros_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_wlbusiness_react-native--canary.845.6859005179.ts'>shadow_sbermarket_wlbusiness_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberonline_react-native--canary.845.6859005179.ts'>shadow_sberonline_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberprime_react-native--canary.845.6859005179.ts'>shadow_sberprime_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_stylesSalute_react-native--canary.845.6859005179.ts'>shadow_stylesSalute_react-native--canary.845.6859005179.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>spacing</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_default_react-native--canary.845.6859005179.ts'>spacing_default_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_b2c_react-native--canary.845.6859005179.ts'>spacing_plasma_b2c_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_web_react-native--canary.845.6859005179.ts'>spacing_plasma_web_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberHealth_react-native--canary.845.6859005179.ts'>spacing_sberHealth_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_business_react-native--canary.845.6859005179.ts'>spacing_sbermarket_business_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_metro_react-native--canary.845.6859005179.ts'>spacing_sbermarket_metro_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_react-native--canary.845.6859005179.ts'>spacing_sbermarket_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_selgros_react-native--canary.845.6859005179.ts'>spacing_sbermarket_selgros_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_wlbusiness_react-native--canary.845.6859005179.ts'>spacing_sbermarket_wlbusiness_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberonline_react-native--canary.845.6859005179.ts'>spacing_sberonline_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberprime_react-native--canary.845.6859005179.ts'>spacing_sberprime_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_stylesSalute_react-native--canary.845.6859005179.ts'>spacing_stylesSalute_react-native--canary.845.6859005179.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>typo</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.845.6859005179.swift'>typo_mage_ios-swift--canary.845.6859005179.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.845.6859005179.kt'>typo_mage_kotlin--canary.845.6859005179.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.845.6859005179.ts'>typo_mage_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.845.6859005179.swift'>typo_plasma_ios-swift--canary.845.6859005179.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.845.6859005179.kt'>typo_plasma_kotlin--canary.845.6859005179.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.845.6859005179.ts'>typo_plasma_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.845.6859005179.swift'>typo_ruler_ios-swift--canary.845.6859005179.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.845.6859005179.kt'>typo_ruler_kotlin--canary.845.6859005179.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.845.6859005179.ts'>typo_ruler_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.845.6859005179.swift'>typo_sage_ios-swift--canary.845.6859005179.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.845.6859005179.kt'>typo_sage_kotlin--canary.845.6859005179.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.845.6859005179.ts'>typo_sage_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.845.6859005179.swift'>typo_sbermarket_ios-swift--canary.845.6859005179.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.845.6859005179.kt'>typo_sbermarket_kotlin--canary.845.6859005179.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.845.6859005179.ts'>typo_sbermarket_react-native--canary.845.6859005179.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.845.6859005179.swift'>typo_soulmate_ios-swift--canary.845.6859005179.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.845.6859005179.kt'>typo_soulmate_kotlin--canary.845.6859005179.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.845.6859005179.ts'>typo_soulmate_react-native--canary.845.6859005179.ts</a><br>
      </blockquote>
    </details>
  </blockquote>
</details>

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-asdk@0.4.1-canary.845.6859005179.0
  npm install @salutejs/plasma-b2c@1.245.1-canary.845.6859005179.0
  npm install @salutejs/plasma-new-hope@0.12.1-canary.845.6859005179.0
  npm install @salutejs/plasma-web@1.245.1-canary.845.6859005179.0
  # or 
  yarn add @salutejs/plasma-asdk@0.4.1-canary.845.6859005179.0
  yarn add @salutejs/plasma-b2c@1.245.1-canary.845.6859005179.0
  yarn add @salutejs/plasma-new-hope@0.12.1-canary.845.6859005179.0
  yarn add @salutejs/plasma-web@1.245.1-canary.845.6859005179.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-845/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-845/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-845/b2c-storybook/
web storybook: http://plasma.sberdevices.ru/pr/pr-845/web-storybook/
new-hope storybook: http://plasma.sberdevices.ru/pr/pr-845/new-hope-storybook/
asdk storybook: http://plasma.sberdevices.ru/pr/pr-845/asdk-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-845/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-845/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-845/b2c-storybook/
web storybook: http://plasma.sberdevices.ru/pr/pr-845/web-storybook/
new-hope storybook: http://plasma.sberdevices.ru/pr/pr-845/new-hope-storybook/
asdk storybook: http://plasma.sberdevices.ru/pr/pr-845/asdk-storybook/

File: packages/plasma-new-hope/.storybook/main.ts [hunk 1]
```diff
@@ -1,32 +1,43 @@
...
             build: {
-                ...config.build,
                 sourcemap: false,
             },
             plugins: [
-                ...config.plugins,
                 linaria({
                     exclude: ['../../../'],
+                    babelOptions: {
+                        presets: ['@babel/preset-typescript', '@babel/preset-react'],
```
Comment (kayman233) [inline packages/plasma-new-hope/.storybook/main.ts:35]: 👍

File: packages/plasma-new-hope/src/examples/plasma_b2c/components/Calendar/Calendar.stories.tsx [hunk 2]
```diff
@@ -25,9 +28,14 @@ export default {
...
-} as ComponentMeta<typeof CalendarBase>;
+};
 
-const onChangeValue = action('onChangeValue');
+export default meta;
+
+type CalendarProps = ComponentProps<typeof Calendar>;
+type CalendarBaseProps = ComponentProps<typeof CalendarBase>;
+type CalendarDoubleProps = ComponentProps<typeof CalendarDouble>;
+type CalendarBaseRangeProps = ComponentProps<typeof CalendarBaseRange>;
```
Comment (kayman233) [inline packages/plasma-new-hope/src/examples/plasma_b2c/components/Calendar/Calendar.stories.tsx:38]: [уточнение]
А почему мы так типы берем? В самом календаре(и в других компонентах) они не экспортятся?

Comment (TitanKuzmich) [reply packages/plasma-new-hope/src/examples/plasma_b2c/components/Calendar/Calendar.stories.tsx:38]: Так в доке сторибука, вроде бы написано

Comment (Yakutoc) [reply packages/plasma-new-hope/src/examples/plasma_b2c/components/Calendar/Calendar.stories.tsx:38]: @kayman233 Можно и так, и так сделать. 

Однако в самом файле `import type { } from './Calendar';` нет re-export для props.

Screenshot 2023-11-14 at 14 18 02

Поэтому пошел по пути наименьшего сопротивления. 

В целом этот момент дискуссионный и можно обсудить различные варианты )

**Review by kayman233 — APPROVED** [review]

File: packages/plasma-new-hope/src/examples/plasma_b2c/components/Calendar/Calendar.stories.tsx (hunk 2 above)
Comment (Yakutoc) [reply packages/plasma-new-hope/src/examples/plasma_b2c/components/Calendar/Calendar.stories.tsx:38]: @kayman233 

Так же я `ComponentProps` использую когда у `StoryObj` не получается понять какие props надо вытащить 

Screenshot 2023-11-14 at 14 32 31

А вот поведение с `ComponentProps`

Screenshot 2023-11-14 at 14 32 10

**Review by TitanKuzmich — APPROVED** [review]

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-asdk@0.5.0-dev.0, @salutejs/plasma-b2c@1.246.0-dev.0, @salutejs/plasma-hope@1.244.0-dev.0, @salutejs/plasma-new-hope@0.13.0-dev.0, @salutejs/plasma-temple-docs@0.183.0-dev.0, @salutejs/plasma-ui-docs@0.232.0-dev.0, @salutejs/plasma-web-docs@0.200.0-dev.0, @salutejs/plasma-web@1.246.0-dev.0, @salutejs/plasma-website@0.208.0-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-asdk@0.5.0-dev.0, @salutejs/plasma-b2c@1.246.0-dev.0, @salutejs/plasma-hope@1.244.0-dev.0, @salutejs/plasma-new-hope@0.13.0-dev.0, @salutejs/plasma-temple-docs@0.183.0-dev.0, @salutejs/plasma-ui-docs@0.232.0-dev.0, @salutejs/plasma-web-docs@0.200.0-dev.0, @salutejs/plasma-web@1.246.0-dev.0, @salutejs/plasma-website@0.208.0-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-asdk@0.5.0-dev.0, @salutejs/plasma-b2c@1.246.0-dev.0, @salutejs/plasma-hope@1.244.0-dev.0, @salutejs/plasma-new-hope@0.13.0-dev.0, @salutejs/plasma-temple-docs@0.183.0-dev.0, @salutejs/plasma-ui-docs@0.232.0-dev.0, @salutejs/plasma-web-docs@0.200.0-dev.0, @salutejs/plasma-web@1.246.0-dev.0, @salutejs/plasma-website@0.208.0-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-asdk@0.5.0-dev.0, @salutejs/plasma-b2c@1.246.0-dev.0, @salutejs/plasma-hope@1.244.0-dev.0, @salutejs/plasma-new-hope@0.13.0-dev.0, @salutejs/plasma-temple-docs@0.183.0-dev.0, @salutejs/plasma-ui-docs@0.232.0-dev.0, @salutejs/plasma-web-docs@0.200.0-dev.0, @salutejs/plasma-web@1.246.0-dev.0, @salutejs/plasma-website@0.208.0-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-asdk@0.5.0-dev.0, @salutejs/plasma-b2c@1.246.0-dev.0, @salutejs/plasma-hope@1.244.0-dev.0, @salutejs/plasma-new-hope@0.13.0-dev.0, @salutejs/plasma-temple-docs@0.183.0-dev.0, @salutejs/plasma-ui-docs@0.232.0-dev.0, @salutejs/plasma-web-docs@0.200.0-dev.0, @salutejs/plasma-web@1.246.0-dev.0, @salutejs/plasma-website@0.208.0-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-asdk@0.5.0-dev.0, @salutejs/plasma-b2c@1.246.0-dev.0, @salutejs/plasma-hope@1.244.0-dev.0, @salutejs/plasma-new-hope@0.13.0-dev.0, @salutejs/plasma-temple-docs@0.183.0-dev.0, @salutejs/plasma-ui-docs@0.232.0-dev.0, @salutejs/plasma-web-docs@0.200.0-dev.0, @salutejs/plasma-web@1.246.0-dev.0, @salutejs/plasma-website@0.208.0-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-asdk@0.5.0-dev.0, @salutejs/plasma-b2c@1.246.0-dev.0, @salutejs/plasma-hope@1.244.0-dev.0, @salutejs/plasma-new-hope@0.13.0-dev.0, @salutejs/plasma-temple-docs@0.183.0-dev.0, @salutejs/plasma-ui-docs@0.232.0-dev.0, @salutejs/plasma-web-docs@0.200.0-dev.0, @salutejs/plasma-web@1.246.0-dev.0, @salutejs/plasma-website@0.208.0-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-asdk@0.5.0-dev.0, @salutejs/plasma-b2c@1.246.0-dev.0, @salutejs/plasma-hope@1.244.0-dev.0, @salutejs/plasma-new-hope@0.13.0-dev.0, @salutejs/plasma-temple-docs@0.183.0-dev.0, @salutejs/plasma-ui-docs@0.232.0-dev.0, @salutejs/plasma-web-docs@0.200.0-dev.0, @salutejs/plasma-web@1.246.0-dev.0, @salutejs/plasma-website@0.208.0-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-asdk@0.5.0-dev.0, @salutejs/plasma-b2c@1.246.0-dev.0, @salutejs/plasma-hope@1.244.0-dev.0, @salutejs/plasma-new-hope@0.13.0-dev.0, @salutejs/plasma-temple-docs@0.183.0-dev.0, @salutejs/plasma-ui-docs@0.232.0-dev.0, @salutejs/plasma-web-docs@0.200.0-dev.0, @salutejs/plasma-web@1.246.0-dev.0, @salutejs/plasma-website@0.208.0-dev.0 🚀

**Pull request merged**